.hemtt/missions/dui.vr/mission.sqm

```diff
@@ -311,13 +303,161 @@ class CustomAttributes
 							class data
 							{
 								singleType="ARRAY";
+								class value
+								{
+									items=2;
+									class Item0
+									{
+										class data
+										{
+											singleType="STRING";
+											value="diwako_dui_linecompass_waypointcolor";
+										};
+									};
+									class Item1
+									{
+										class data
+										{
+											singleType="STRING";
+											value="diwako_dui_linecompass_acefingeringcolor";
+										};
+									};
+								};
```

Comment:
what is going on here, should we just reset the mission.sqm?

Reply:
that is just the CBA settings Hash. when you add settings they get added here if you load the mission

Reply:
I would prefer these being tested using the `cba_settings.sqf` file in the test mission. Makes it imo easier to see and change without having to go into the menu all the time.

Reply:
there are no settings in the mission. I didn't touch that. it just stored the default values of the new settings in there. i can just revert the file. but then every time you open it again it will be added again. so I don't really care